terraform/aws/components/gpc-consumer/locals_env_variables.tf

```diff
@@ -0,0 +1,124 @@
+locals {
+  environment_variables = concat(var.gpc-consumer_environment_variables,[
+    {
+      name  = "GPC_CONSUMER_SERVER_PORT"
+      value = var.gpc-consumer_service_container_port
+    },
+    {
+      name  = "GPC_CONSUMER_ROOT_LOGGING_LEVEL"
+      value = var.gpc-consumer_root_log_level
+    },
+    {
+      name  = "GPC_CONSUMER_ROOT_LOGGING_LEVEL"
+      value = var.gpc-consumer_log_level
+    },
+    {
+      name  = "GPC_CONSUMER_LOGGING_FORMAT"
+      value = var.gpc-consumer_logging_format
+    },
+    {
+      name  = "GPC_CONSUMER_URL"
+      value = "http://${module.gpc-consumer_ecs_service[0].loadbalancer_dns_name}:${var.gpc-consumer_service_container_port}"
```

Comment:
as the gpc-consumer module does not have `count` on it the [0] index is not needed here.

Reply:
removed [0] index 

    {
      name  = "GPC_CONSUMER_URL"
      value = "http://${module.gpc-consumer_ecs_service.loadbalancer_dns_name}:${var.gpc-consumer_service_container_port}"
    },